style.css

```diff
@@ -23,40 +18,104 @@ body {
   flex-direction: column;
   justify-content: flex-start;
   align-items: center;
-  margin: 2%;
+  margin: 0;
 }
 
-.content {
-  width: 50%;
+h1,
+h2 {
+  margin: 0;
+}
+
+section {
+  padding: 3rem 0;
+}
+
+#librarySection {
+  width: 98%;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  margin-left: 2px;
-  margin-right: 2px;
+  padding-right: 1rem;
+  padding-left: 1rem;
+  gap: 2rem;
+}
+
+form * {
+  display: flex;
+  box-sizing: border-box;
 }
 
-main {
+.book-section {
   display: flex;
   flex-direction: column;
-  justify-content: center;
+  width: 97%;
+  padding: 1rem;
+  font-weight: lighter;
+}
+
+.book-section * {
+  font-size: 0.8rem;
+}
+
+.date {
+  align-self: flex-end;
+}
+
+.navbar {
+  width: auto;
+  height: max-content;
+  display: flex;
```

Comment:
- [ ] You have used `display: flex` 11 times which means you are repeating yourself alot, this falls under the category of bad practice of CSS. I would recommend you create a class with display:flex on it and give it to any HTML that needs it 
![image](https://user-images.githubusercontent.com/62705947/219072384-a8f5b633-f738-42c7-a730-0d7491d2d070.png)